cfme/automate/service_dialogs.py

```diff
@@ -113,16 +113,14 @@ class DialogCollection(BaseCollection):
     tree_path = ['All Dialogs']
     ENTITY = Dialog
 
-    def create(self, label=None, description=None, submit=False, cancel=False):
+    def create(self, label=None, description=None, submit=True, cancel=True):
```

Comment:
**REQUIRED**: Can we rename ```submit``` and ```cancel``` as they sound like they refer to the create operation

Reply:
The button names in dialog form are submit and cancel. Cannot be changed

Reply:
I can change it to submit_btn but I have come to know that we are removing btn from everywhere